src/index.ts

```diff
@@ -43,6 +43,7 @@ export default function connect(
 
     socket.onopen = () => {
       open = true
+      forcedClose = false
```

Comment:
This isn't necessary I think? After the websocket completes/errors the `observer` and scope tied to it will be destroyed and a reconnect will create a new `observer`/scope with a new `forcedClose` no?